conf/test_full.config

```diff
@@ -10,6 +10,8 @@
 ----------------------------------------------------------------------------------------
 */
 
+cleanup = true
+
```

Comment:
This should go, otherwise we don't keep results

Reply:
Thanks, but when do we not keep results? Because in nf-core/mag it seems to appear as well: https://github.com/nf-core/mag/blob/66cf53aff834d2a254b78b94fc54cd656b8b7b57/conf/test_full.config#L13 and it seems to keep results on the [website](https://nf-co.re/mag/results) (in case you mean that...).